rift/credentials/creds.go

```diff
@@ -0,0 +1,44 @@
+package credentials
+
+import (
+	"context"
+	"regexp"
+
+	"github.com/rotisserie/eris"
+	"google.golang.org/grpc/credentials"
+)
+
+var (
+	TokenKey = "router_key"
+
+	_ credentials.PerRPCCredentials = &simpleTokenCredential{}
+
+	routerKeyRegexp = regexp.MustCompile(`^[a-zA-Z0-9]{64}$`)
+)
+
+type simpleTokenCredential struct {
```

Comment:
nit: just call it `tokenCredential`, I don't think "simple" really gives us anything here.

Reply:
fixed